config/projects/google-fluentd.rb

```diff
@@ -8,13 +8,13 @@
 description "Google Fluentd: A data collector for Google Cloud Logging"
 
 install_dir     "/opt/google-fluentd"
-build_version   "1.5.9"
+build_version   "1.5.9.grpc.alpha.2"
```

Comment:
I'd imagine we'd want to abstract away the gRPC support in the version name as it would show up during the installation process?